src/components/app.tsx

```diff
@@ -33,55 +31,63 @@ export class App extends React.PureComponent<IProps, IState> {
   }
 
   private renderActivity = () => {
+    const { currentPage } = this.state;
+    let totalPreviousQuestions = 0;
+
+    for (let page = 0; page < (currentPage - 1); page++) {
+      for (let embeddable = 0; embeddable < sampleActivity.pages[page].embeddables.length; embeddable++) {
+        if (!sampleActivity.pages[page].embeddables[embeddable].section) {
+          totalPreviousQuestions++;
+        }
+      }
+    }
+
+    const fullWidth = currentPage!==0 && (sampleActivity.pages[currentPage - 1].layout === "l-responsive");
```

Comment:
```suggestion
    const fullWidth = (currentPage !== 0) && (sampleActivity.pages[currentPage - 1].layout === "l-responsive");
```
consistency

Reply:
Fixed.  And I removed the string constant.